3.6.0 changelog (#3203)
AzureAD/microsoft-identity-web: changelog.md

3.6.0
========

> - Updated to Microsoft.IdentityModel.* 8.4.0

Reviewer: ```suggestion
- Updated to Microsoft.IdentityModel.* 8.3.1
```

- Updated to [MSAL.NET](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet) 4.67.2

## Bug fixes

> - Add checks to protect the internal claims used by Identity Web. See https://github.com/AzureAD/microsoft-identity-web/pull/3131

Reviewer 2: ```suggestion
- Checks that B2C tokens don't contain the claims used by Identity Web to represent the home tenant and object ID (obtained from the UserInfo endpoint). See https://github.com/AzureAD/microsoft-identity-web/pull/3131
```

> - 

Reviewer: ```suggestion
- Remove explicit locking in `OpenIdConnectCachingSecurityTokenProvider`. See Issue [#3078](https://github.com/AzureAD/microsoft-identity-web/issues/3078)

```
